drafts/25-xxx-specifications.txt

```diff
@@ -296,19 +296,25 @@ di07. The non-directive directive
 7 Predefined macros
 ===================
 
+pm00. Macro names beginning with a leading underscore followed by an uppercase
+      letter or a second underscore are reserved to the processor. Reserved
+      names shall not be the subject of a #define or #undef preprocessing
+      directive within a program unit.
+      [Note: Any valid macro name matching the regular expression `/^_[A-Z_].*/`
+      is considered reserved.]
+
 pm01. Any macro name predefined by the implementation shall begin with a
       leading underscore followed by an uppercase letter or a second
       underscore.
 
-pm02. The implementation shall not predefine the macro `__cplusplus`,
+pm02. The processor shall not predefine the macro `__cplusplus`,
```

Comment:
I think we've decided to use Fortran terminology ("the processor", "processor-dependent") instead of C terminology ("the implementation", "implementation-defined").

As such, we should probably apply that rule consistently throughout the document. I believe all the remaining instances appear below in this section.

SIDE-NOTE: C makes a subtle distinction between "unspecified behavior" and "implementation-defined behavior". Fortran seems to generally lump both categories into  "processor-dependent", so we should probably follow that convention.

Reply:
And of course C++ actually has further distinctions between [implementation-defined behavior](https://eel.is/c++draft/intro.defs#defns.impl.defined), [unspecified behavior](https://eel.is/c++draft/intro.defs#defns.unspecified), and [undefined behavior](https://eel.is/c++draft/intro.defs#defns.undefined).

Reply:
> C++ actually has further distinctions between [implementation-defined behavior](https://eel.is/c++draft/intro.defs#defns.impl.defined), [unspecified behavior](https://eel.is/c++draft/intro.defs#defns.unspecified), and [undefined behavior](https://eel.is/c++draft/intro.defs#defns.undefined).

Well sure, all three languages have a concept of undefined behavior, but that's not relevant here.

My point was just that Fortran lacks the subtle distinction between "unspecified"("processor dependent") and "implementation-defined" (no Fortran equivalent)